GameServerLib/GameObjects/GameObject.cs

```diff
@@ -219,6 +222,7 @@ public virtual float GetZ()
 
         public bool IsCollidingWith(IGameObject o)
         {
+            // TODO: That assumes that everything is circle, consider checking other shapes
```

Comment:
I don't remember 100%, but I think they are.

Reply:
You are right. I remember having implemented a specific collision check for "Lasers" (Lux Ult etc.) but somehow it got broken with updates.